packages/sandboxes/hello-world/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@tsconfig/node18/tsconfig.json",
```

Comment:
@petersalas I think we can include a `tsconfig` in our `ai-jsx` package and then people can do `"extends": "ai-jsx/tsconfig"` and get the requisite JSX settings automatically. Maybe we can do that once the dust has settled a bit more.